contracts/mirror_factory/src/contract.rs

```diff
@@ -657,6 +657,10 @@ pub fn migrate_asset(
         return Err(StdError::generic_err("unauthorized"));
     }
 
+    // check token name and symbol validity to prevent instantiation of new token from failing in case they are invalid
+    assert_valid_token_name(name.as_str())?;
```

Comment:
Different from the whitelist operation, in the migration operation, if the instantiation fails, the state would not be reverted (weights from delisted asset removed), so we have to make sure that the instantiation will not fail before executing.